include/net/nrf_cloud_location.h

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef NRF_CLOUD_LOCATION_H_
+#define NRF_CLOUD_LOCATION_H_
+
+/** @file nrf_cloud_location.h
+ * @brief Module to provide nRF Cloud location support to nRF9160 SiP.
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/net/wifi_mgmt.h>
+#include <modem/lte_lc.h>
+#include <net/nrf_cloud.h>
+#include <cJSON.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @defgroup nrf_cloud_location nRF Cloud location
+ * @{
+ */
+
+/** @brief Access points found during a Wi-Fi scan. */
+struct wifi_scan_info {
+	/** Array of access points found during scan. */
+	struct wifi_scan_result *ap_info;
+	/** The number of access points found during scan. */
+	uint16_t cnt;
+};
+
+/** Size of Wi-Fi MAC address string: 2 chars per byte, colon separated */
+#define WIFI_MAC_ADDR_STR_LEN	((WIFI_MAC_ADDR_LEN * 2) + 5)
```

Comment:
```suggestion
#define WIFI_MAC_ADDR_STR_LEN	(WIFI_MAC_ADDR_LEN * 3) - 1
```

Since the number of colons is also determined by `WIFI_MAC_ADDR_LEN`, or am I mistaken on that?

Reply:
This is honestly more of a nit, since the print format is hard-coded anyways